packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -137,7 +138,7 @@ export class Interpreter {
         return;
       }
     }
-    throw new TealError(ERRORS.TEAL.LABEL_NOT_FOUND, {
+    throw new TealError(TEAL_ERRORS.TEAL.LABEL_NOT_FOUND, {
```

Comment:
`TEAL_ERRORS.TEAL.LABEL_NOT_FOUND` has `TEAL` two times. 
Is it possible to have only Runtime Errors (then we won't need `TEAL_ERRORS`, `RUNTIME_ERRORS`), and we could just use `ERRORS.TEAL.LABEL_NOT_FOUND`, `ERRORS_TRANSACTION.TYPE_Err..`).
Basically i am asking if we could use runtime error class only and move teal errors into runtime errors

Reply:
Yes I was also thinking about EXEC, 'teal execution error' 
Renaming to TEAL_ERRORS.EXEC.